### PR TITLE
Add test protocol to validate resolution #6

### PR DESCRIPTION
**Pull request by pombredanne**
This PR provides a base documentation to validate dependency resolution.
 
Signed-off-by: Philippe Ombredanne <pombredanne@nexb.com>

### CODE REVIEW

**Comment by pombredanne** [comment]
@TG1999 Can you help me make it pass the tests?

**Comment by TG1999** [comment]
@pombredanne All green now!

**Comment by pombredanne** [comment]
@TG1999 the steps would benefit a lot from an actual end to end example.I added a starter for this but your review and help to complete this would be awesome!

**Review by TG1999 — APPROVED** [review]
LGTM!

File: docs/source/test-protocol.rst [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+  import json
+  with open("~/tmp/pyinsp-example/codebase/output/deployed-scan-output.json") as f:
+      scancode_data = json.load(f)
+  scancode_purls = []
+  for package in scancode_data["packages"]:
+      if package["purl"] not in scancode_purls:
+          scancode_purls.append(package["purl"])
+  scancode_purls = sorted(scancode_purls)
+  with open("~/tmp/pyinsp-example/codebase/output/scan.txt", "w") as f:
+      f.writelines("\n".join(scancode_purls))
```
Comment (pombredanne) [inline docs/source/test-protocol.rst:271]:
```suggestion
      f.write("\n".join(scancode_purls))
```

**Review by pombredanne — COMMENTED** [review]
LGTM!

**Pull request merged**